moped-editor/src/routes.js

```diff
@@ -20,20 +20,20 @@ import DashboardView from "./views/dashboard/DashboardView";
 export const routes = [
   { path: "/", element: <Navigate to="/moped" /> },
   {
-    path: "moped/session",
+    path: "/moped/session",
     element: <MainLayout />,
     children: [
       { path: "signin", element: <LoginView /> },
       { path: "*", element: <Navigate to="/signin" /> },
     ],
   },
   {
-    path: "moped",
+    path: "/moped",
```

Comment:
this path must be absolute in order for the redirect from `/moped` to `/moped/projects` to work.